R4145/index.html

```diff
@@ -148,9 +148,9 @@
                 <div>Address : No 123,  ***********.</div>
             </div>
             <div class="social_link">
-                <a href="" class="hr">LinkdIn</a>
-                <a href="" class="h">Twitter</a>
-                <a href="" class="h">GitHub</a>
+                <a href="https://www.linkedin.com/in/chamith-sadeepa-kulathung" target="_blank" class="hr">LinkdIn</a>
```

Comment:
You're used a class name called hr. but it seems that there is no .hr class styles.